SingularityService/src/test/java/com/hubspot/singularity/SingularityHistoryPurgerTest.java

```diff
@@ -0,0 +1,154 @@
+package com.hubspot.singularity;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.skife.jdbi.v2.DBI;
+import org.skife.jdbi.v2.Handle;
+
+import com.google.common.base.Optional;
+import com.google.inject.Inject;
+import com.google.inject.Provider;
+import com.hubspot.singularity.config.HistoryPurgingConfiguration;
+import com.hubspot.singularity.data.history.HistoryManager;
+import com.hubspot.singularity.data.history.SingularityHistoryPurger;
+
+import liquibase.Liquibase;
+import liquibase.database.Database;
+import liquibase.database.DatabaseFactory;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.resource.FileSystemResourceAccessor;
+
+public class SingularityHistoryPurgerTest extends SingularitySchedulerTestBase {
+
+  @Inject
+  protected Provider<DBI> dbiProvider;
+
+  @Inject
+  protected HistoryManager historyManager;
+
+  public SingularityHistoryPurgerTest() {
+    super(true);
+  }
+
+  @Before
+  public void createTestData() throws Exception {
+    Handle handle = dbiProvider.get().open();
+
+    Database database = DatabaseFactory.getInstance().findCorrectDatabaseImplementation(new JdbcConnection(handle.getConnection()));
+
+    Liquibase liquibase = new Liquibase("singularity_test.sql", new FileSystemResourceAccessor(), database);
```

Comment:
you should use `../mysql/migrations.sql` instead so that we don't have to update the schema in two different places